modules/candidate_list/README.md

```diff
@@ -24,30 +25,29 @@ to other relevant LORIS modules where applicable.
 ## Permissions
 
 Accessing the `candidate_list` module requires either the `data_entry`
-permission and at least one study site, or the `access_all_profiles`
-permission.
+permission (_"Access Profile: View/Create Candidates and Timepoints - Own Sites"_)
+and at least one study site, or the `access_all_profiles`
+permission (_"Access Profile: View Candidates and Timepoints - All Sites"_).
 
 Users with the `access_all_profiles` permission can see every
 candidate in LORIS, while users with only the `data_entry` permission
-can exclusively see candidates at study sites where they are affiliated.
+can exclusively see candidates at study sites with which they are affiliated.
 
 ## Configurations
 
-The `useEDC` configuration variable has a similar function for the
-"EDC" ("EDC" stands for "Expected Date Of Confinement" which refers
-to the pregnancy due date) column.
+The `useEDC` variable (_"Use EDC"_) can be configured to show or hide the
+"EDC" column in the data table. "EDC" stands for "Expected Date Of Confinement",
+which refers to the pregnancy due date. This variable can be turned on or off
+in the Configurations module.
 
 ## Interactions with LORIS
 
 The "Scan Done" column indicates whether or not a scan was performed
-for that candidate. When a scan was performed (i.e. it has the value
-"Y"), clicking the table cell links to the `imaging_browser` module
-prefiltered for that candidate.
-
-Clicking on the cell in the PSCID column takes the user to the
-`timepoint_list` page for that candidate. The link is only clickable
-if the user does *not* have the `data_entry` permission. For users
-who have the `data_entry` permission, a separate form appears for
-the user to type the CandID/PSCID combo in order to access the
-candidate.  This is intended to prevent data entry errors caused
-by accidentally clicking on the wrong row before doing data entry.
+for that candidate. If a scan was performed, the value "Y" in the column
+will contain a link to the `imaging_browser` module prefiltered for that candidate.
+
+Clicking on the link in the PSCID column takes the user to the
+`timepoint_list` page for that candidate. For users who do *not*
+have the `access_all_profiles` permission, an _Open Profile_ form can be
+used to access a candidate from a restricted site if the correct CandID
+and PSCID are provided.
```

Comment:
@driusan please see last commit. this section needs to be changed because the workflow changed, and so the information is no longer correct.

for context, PSCID links are now always clickable, regardless of permission. this is because the workflow for using the 'Open Profile' form was changed. The form was previously used when the PSCID was not clickable. Now, the form is used only to access restricted candidates (from a site with which you are not affiliated).
this change corrects the permissions and the intended use of the Open Profile form. The data entry error reasoning is no longer relevant.

(i *believe* this is the case, but i might be wrong?)